introduction.md

```diff
@@ -1,3 +1,8 @@
 I'm Daylan. I am kind of enjoying working from the command line. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?